cmd/expiration-mailer/main.go

```diff
@@ -44,16 +44,75 @@ type regStore interface {
 	GetRegistration(ctx context.Context, req *sapb.RegistrationID, _ ...grpc.CallOption) (*corepb.Registration, error)
 }
 
+// limiter tracks how many mails we've sent to a given address in a given day.
```

Comment:
A restart of the expiration mailer process would clear this in-memory store out too.